recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.imx.bb

```diff
@@ -63,6 +66,7 @@ PACKAGECONFIG[hls]             = "-Dhls=enabled -Dhls-crypto=nettle,-Dhls=disabl
 # to -base yet. They depend on the gstgl library in -base, so we do
 # not add GL dependencies here, since these are taken care of in -base.
 PACKAGECONFIG[gl]              = "-Dgl=enabled,-Dgl=disabled,"
+PACKAGECONFIG[gles2]           = ",,virtual/libgles2"
```

Comment:
also AFAIK the plugins use GstGLAPI instead of directly the opengl API, so the dependency should not be necessary at all.

Reply:
Have you prepare, and test, a PR with the proposed change?